documentcloud/entities/serializers.py

```diff
@@ -0,0 +1,30 @@
+from .models import Entity
+from rest_framework import serializers
+
+
+class EntitySerializer(serializers.HyperlinkedModelSerializer):
+    # entities = serializers.PrimaryKeyRelatedField(
+    #     many=True, queryset=Entity.objects.all()
+    # )
+    # Use document's serializer
```

Comment:
The serializer needs some work - lets review it together and compare to some of the other serializers we have